workflow/common/convert.go

```diff
@@ -8,22 +8,21 @@ import (
 )
 
 func ConvertCronWorkflowToWorkflow(cronWf *wfv1.CronWorkflow) *wfv1.Workflow {
-	wf := toWorkflow(cronWf.TypeMeta, cronWf.ObjectMeta, cronWf.Spec.WorkflowSpec)
```

Comment:
is this covered by existing tests?

Reply:
Yes it is. I added an extra test as well